client/coral-admin/src/routes/Moderation/components/Comment.css

```diff
@@ -61,6 +61,7 @@
   line-height: 1.5;
   font-weight: 300;
   margin-bottom: 8px;
+  word-break: break-word;
```

Comment:
Even better break is overflow-wrap: break-word; which only breaks word if absolute necessary.

See note in, which explains the contrast to word-break.
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap